libraries/botframework-connector/src/connectorApi/models/mappers.ts

```diff
@@ -956,6 +956,28 @@ export const ConversationsResult: msRest.CompositeMapper = {
   }
 };
 
+export const ExpectedReplies: msRest.CompositeMapper = {
```

Comment:
Not sure this is needed as the activities will come in a response to BotframeworkClient.postActivity()... but otherwise it looks good!